test/Markdown.MAML.Test/Transformer/ParserAndTransformerTestsV2.cs

```diff
@@ -70,8 +69,73 @@ public void TransformCommandWithExtraLine()
 
 ");
             MamlCommand mamlCommand = NodeModelToMamlModelV2(doc).First();
-            Assert.Equal(mamlCommand.Name, "Add-Member");
-            Assert.Equal(mamlCommand.Synopsis, "Adds custom properties and methods to an instance of a Windows PowerShell object.");
+            Assert.Equal("Add-Member", mamlCommand.Name);
+            Assert.Equal("Adds custom properties and methods to an instance of a Windows PowerShell object.", mamlCommand.Synopsis.Text);
```

Comment:
Thanks for fixing these parameter order!
I made a mistake early in development and copy-pasted it everywhere